dataconnect/app/src/main/java/com/google/firebase/example/dataconnect/ui/components/ReviewCard.kt

```diff
@@ -55,10 +56,23 @@ fun ReviewCard(
                 modifier = Modifier.padding(bottom = 8.dp)
             ) {
                 Text(
-                    text = SimpleDateFormat(
-                        "dd MMM, yyyy",
-                        Locale.getDefault()
-                    ).format(date),
+                    text =
+                        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
+                            val dateFormatter = DateTimeFormatter.ofPattern("MMM d, yyyy", Locale.getDefault())
+                            date.toJavaLocalDate().format(dateFormatter)
+                        } else {
+                            val parseableDateString = date.run {
+                                val year = "$year".padStart(4, '0')
+                                val month = "$month".padStart(2, '0')
+                                val day = "$day".padStart(2, '0')
+                                "$year-$month-$day"
+                            }
+                            val dateParser = SimpleDateFormat("y-M-d", Locale.US)
+                            val parsedDate = dateParser.parse(parseableDateString) ?:
+                              throw Exception("INTERNAL ERROR: unparseable date string: $parseableDateString")
+                            val dateFormatter = SimpleDateFormat("MMM d, yyyy", Locale.getDefault())
```

Comment:
Same comment as above

```suggestion
                            val dateFormatter = SimpleDateFormat("dd MMM, yyyy", Locale.getDefault())
```

Reply:
Done.